meta-k8s-setup/recipes-k8s-configuration/k8s-configuration/files/zap-ceph-disks.service

```
@@ -0,0 +1,13 @@
+[Unit]
+Description=zap ceph disks
+DefaultDependencies=no
+Conflicts=shutdown.target
+After=local-fs.target
+Before=sysinit.target shutdown.target
+
+[Service]
+Type=oneshot
+ExecStart=ash -c 'udevadm settle; for DISK in $$(lsblk -p -o NAME,FSTYPE | grep LVM2_member | awk \'{print $$1}\'); do wipefs --all $$DISK; sgdisk --zap-all $$DISK; dd if=/dev/zero of=$$DISK bs=1M count=10; sync; done'
```

Perhaps we should use `&&` 🤔:
```suggestion
ExecStart=ash -c 'udevadm settle && for DISK in $$(lsblk -p -o NAME,FSTYPE | grep LVM2_member | awk \'{print $$1}\'); do wipefs --all $$DISK && sgdisk --zap-all $$DISK && dd if=/dev/zero of=$$DISK bs=1M count=10 && sync; done'
```

---

Fixed with `noerrexit`.